### PR TITLE
Custom Resource example

### PR DESCRIPTION
**Pull request by akanso**
targeting issue: #538

This example creates, lists, Gets, updates and deletes a custom resource.

fixes the issue of author name/email in #539 
The rest is exactly the same

### CODE REVIEW

**Comment by akanso** [comment]
Windows build failed because =>

`C:\Users\runneradmin\.nuget\packages\nerdbank.gitversioning\3.3.37\build\Nerdbank.GitVersioning.targets(115,5): error MSB4018: System.IO.IOException: The process cannot access the file 'D:\a\_temp\_runner_file_commands\set_env_e54b7874-714f-475a-9bab-e5ee7ae37d54' because it is being used by another process. `

perhaps we should re-run the pipeline...

**Review by tg123 — APPROVED** [review]

File: examples/customResource/customResource.sln [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿
```
[the remark this answers is not in the document]
Comment (brendandburns) [reply examples/customResource/customResource.sln:1]: agree.

File: examples/customResource/utils.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+using k8s.Models;
```
Comment (brendandburns) [inline examples/customResource/utils.cs:1]: rename this file to `Utils.cs`

**Comment by brendandburns** [comment]
Couple of minor comments and then this is good to merge.

**Comment by akanso** [comment]
In the Ubuntu test: `Failed!  - Failed:     3, Passed:   151, Skipped:     1, Total:   155, Duration: 2 m 31 s - `

However on my machine:
```
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.5 LTS
Release:        18.04
Codename:       bionic
```
the test passes:
```
Passed!  - Failed:     0, Passed:   147, Skipped:     1, Total:   148, Duration: 2 m 10 s - /home/azureuser/csharp/tests/KubernetesClient.Tests/bin/Debug/netcoreapp2.1/KubernetesClient.Tests.dll (netcoreapp2.1)
```

File: examples/customResource/utils.cs (hunk 2 above)
Comment (akanso) [reply examples/customResource/utils.cs]: Done! @brendandburns

**Comment by brendandburns** [comment]
Must have been a flake because the tests now look ok.

/lgtm
/approve

Thanks for the patience!

**Pull request merged**